resources/views/brands/show.blade.php

```diff
@@ -0,0 +1,27 @@
+@extends('rapidez::layouts.app')
+
+@section('title', $meta_title->value() ?: $title)
+@section('description', $meta_description)
+@push('head')
+    @foreach(Statamic::tag('alternates')->params(compact('page')) as $lang => $url)
+        <link rel="alternate" hreflang="{{ $lang }}" href="{{ $url }}" />
+    @endforeach
+@endpush
+
+@php($brandAttribute = \Rapidez\Core\Models\Attribute::find(config('rapidez.statamic.runway.brand_attribute_id')))
+
+@section('content')
+    @includeWhen(!$is_homepage, 'rapidez-statamic::breadcrumbs')
+
+    @if ($title)
+        <h1 class="font-bold text-3xl mb-5">{{ $title }}</h1>
+    @endif
+    @if ($description)
+        <div class="prose prose-green max-w-none">
+            @content($description)
+        </div>
+    @endif
```

Comment:
This doesn't exist in the blueprint? Maybe just move the page builder include here?

Reply:
I added it to the blueprint. I'd say we only want a description above the product listing. If we want to add other content you'd want it below the product grid.